### PR TITLE
Implement URL rewrite (also, re-indent nginx file…)

### LINKED ISSUE #61: URL rewrite not implemented
OpenSondage usually works with URL rewrite. That means that :
```
https://myserver.tld/sondage/poll_name
```
should redirect to 
```
https://myserver.tld/sondage/studs.php?poll=poll_name
```

It's implemented by their .htaccess file, found here : https://github.com/framasoft/framadate/blob/github/htaccess.txt
```
  RewriteRule ^([a-zA-Z0-9-]+)$ studs.php?poll=$1 [L]
  RewriteRule ^([a-zA-Z0-9-]+)/action/([a-zA-Z_-]+)/(.+)$ studs.php?poll=$1&$2=$3
  RewriteRule ^([a-zA-Z0-9-]+)/vote/([a-zA-Z0-9]{16})$ studs.php?poll=$1&vote=$2
  RewriteRule ^([a-zA-Z0-9-]{24})/admin$ adminstuds.php?poll=$1
  RewriteRule ^([a-zA-Z0-9-]{24})/admin/vote/([a-zA-Z0-9]{16})$ adminstuds.php?poll=$1&vote=$2
  RewriteRule ^([a-zA-Z0-9-]{24})/admin/action/([a-zA-Z_-]+)(/(.+))?$ adminstuds.php?poll=$1&$2=$4
```
It should ideally be implemented by the nginx config file too. I tried it successfully, gonna do a PR soon.

### PR DESCRIPTION
**Pull request by Salamandar**
Fixes #61

### CODE REVIEW

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
+    # Admin requires the private poll ID, 24-char long
+    rewrite "^__PATH__/([a-zA-Z0-9-]{24})/admin$"                               "__PATH__/adminstuds.php?poll=$1";
+    rewrite "^__PATH__/([a-zA-Z0-9-]{24})/admin/vote/([a-zA-Z0-9]{16})$"        "__PATH__/adminstuds.php?poll=$1&vote=$2";
+    rewrite "^__PATH__/([a-zA-Z0-9-]{24})/admin/action/([a-zA-Z_-]+)(/(.+))?$"  "__PATH__/adminstuds.php?poll=$1&$2=$4";
+
+    index index.php;
+    try_files $uri $uri/ /index.php;
+    location ~ [^/]\.php(/|$) {
+        fastcgi_split_path_info ^(.+?\.php)(/.*)$;
+        fastcgi_pass unix:/var/run/php/php7.0-fpm-__NAME__.sock;
```
Comment (ericgaspar) [inline conf/nginx.conf:25]:
```suggestion
        fastcgi_pass unix:/var/run/php/php__PHPVERSION__-fpm-__NAME__.sock;
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
As we have now PHP set to 7.3 in _common.sh

**Comment by Salamandar** [comment]
I think that's good now, the requested changes were merge before I had time to read them ^^'

**Comment by ericgaspar** [comment]
package 'php7.3-fpdf' has no installation candidate...
we may need to set it as 'php-fpdf'

**Comment by Salamandar** [comment]
We can do another pull request for that :D
Maybe other packages have the same issue.

**Comment by ericgaspar** [comment]
This is failing the CI tests...

**Comment by Salamandar** [comment]
My changes aren't related to php installation, so I don't understand.
Oh, but i'm not merging on Master but Testing ! Testing should be failing on CI too I guess.

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**